customer-service/src/main/resources/application.properties

```diff
@@ -5,10 +5,16 @@ logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.springframework.retry=TRACE
 
 
-spring.datasource.url=jdbc:mysql://${DOCKER_HOST_IP:localhost}/eventuate
+spring.datasource.url=jdbc:mysql://localhost/eventuate
```

Comment:
There is no point in making this change.

Reply:
Why? I removed every mention of DOCKER_HOST_IP. Why leave it here?

Reply:
It's not a big deal but it leaves open the option of using DOCKER_HOST_IP in some unusual situation.

Reply:
Should I replace localhost by ${DOCKER_HOST_IP:localhost} everywhere in each project?